src/rng.cpp

```diff
@@ -0,0 +1,137 @@
+/*
+ * rng.cpp
+ *
+ *  Created on: 25 Mar 2020
+ *      Author: gc1610
+ */
+
+#include <Rcpp.h>
+
+float genexp(float av) {
+	return R::rexp(av);
+}
+
+float gennor(float av, float sd) {
+	return R::rnorm(av, sd);
+}
+
+float genunf(float low, float high) {
+	return R::runif(low, high);
+}
+
+
+void setgmn(float *meanv,float *covm,long p,float *parm)
```

Comment:
What is the licence of this code? Do we really want this only on `float` not `double` given that the underlying RNG is generating us doubles?